static/app/components/events/aiAutofix/index.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import {AiAutofix} from 'sentry/components/events/aiAutofix';
+import {useAiAutofix} from 'sentry/components/events/aiAutofix/useAiAutofix';
+import {Group} from 'sentry/types';
+
+jest.mock('sentry/components/events/aiAutofix/useAiAutofix', () => ({
+  useAiAutofix: jest.fn(() => ({
+    autofixData: null,
+    triggerAutofix: jest.fn(),
+    additionalContext: '',
+    setAdditionalContext: jest.fn(),
+    error: null,
+    isError: false,
+    isPolling: false,
+  })),
+}));
+
+const group = {
+  /* Mock group data */
+  id: 1,
+} as unknown as Group;
+
+describe('AiAutofix', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('renders the Banner component when autofixData is null', () => {
+    render(<AiAutofix group={group} />);
+
+    expect(screen.getByText('AI Autofix')).toBeInTheDocument();
+  });
+
+  it('renders the FixResult component when autofixData is present', () => {
+    (useAiAutofix as jest.Mock).mockReturnValue({
```

Comment:
It would be better to test without mocking this out, which shouldn't be too difficult. If this makes a request, you can add a mock response with MockApiClient (see [docs](https://develop.sentry.dev/frontend/network-requests/#testing-components--hooks-with-network-requests)). You'll also want to put your expectations below inside of a `waitFor()`